<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>182</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="qlinedit_ip" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="qlinedit_port" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>10</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpushbutton_OK" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
